content/11.interpretation.md

```diff
@@ -10,11 +10,10 @@ While we know that accurately predicting an outcome doesn't imply learning the c
 A poignant example of this lesson is [@tag:predicting-pneumonia-mortality; @doi:10.1145/2783258.2788613].
 In this study, the authors evaluated the capacities of several models to predict the probability of death for patients admitted to an intensive care unit with pneumonia.
 Unsurprisingly, the neural network model achieved the best predictive accuracy.
-However, after fitting a rule-based model, the authors discovered that the hospital data implied the rule "HasAsthma(x) => LowerRisk(x)".
+However, after fitting a rule-based model, the authors discovered that the hospital data implied the rule `HasAsthma(x) => LowerRisk(x)`.
 This rule contradicts medical understanding - having asthma doesn't make pneumonia better!
 This rule was supported by the data (pneumonia patients with a history of pneumonia tended to receive more aggressive care), so the neural network also learned to make predictions according to this rule.
 Guiding treatment decisions according to the predictions of the neural network would have been disastrous, even though the neural network had high predictive accuracy.
 
 To trust deep learning models, we must combine knowledge of the training data ([Tip 4](#know-your-problem)) with inspection of the model ([Tip 8](#blackbox)).
-By probing data domains where the model succeeds and contrasting with domains where the model fails, we can identify the internal logic and deduce scientific conclusions.
-In this way, we can move beyond fitting predictive models toward building understanding.
+To move beyond fitting predictive models toward building understanding and deduce scientific conclusions, probe data domains where your model succeeds and contrast them with domains where your model fails in order to identify your model's internal logic.
```

Comment:
The "internal logic" was there before, but that phrase could risk teaching readers to over-interpret their models as cautioned against in the first paragraph.